crates/biome_js_analyze/src/analyzers/nursery/use_import_restrictions.rs

```diff
@@ -137,7 +137,7 @@ fn get_restricted_import(module_path: &TokenText) -> Option<ImportRestrictionsSt
                 // We pop the index file because it shouldn't count as a path,
                 // component, but we store the file name so we can add it to
                 // both the reported path and the suggestion.
-                index_filename = path_parts.last().cloned();
+                index_filename = path_parts.last().copied();
```

Comment:
This seems like a mistake.

Reply:
I think it is correct because we copy the string slice which is a copyable type?

You can find more details about `cloned` vs `copied` [here](https://stackoverflow.com/questions/58413583/difference-between-copied-and-cloned-on-rust-iterators).

Reply:
OK, so this was the result of which rule again?

Reply:
[cloned_instead_of_copied](https://rust-lang.github.io/rust-clippy/master/#/cloned_instead_of_copied).

Do you think it is too pedantic?

Reply:
Nope, I am 🤣